### ISSUE TITLE
Docs: Explicit note that it only works with Node. Not in the browser.

### ISSUE THREAD

**Issue by y0nd0** [grimmdude/MidiWriterJS#83]
It would be helpful if there is a prominent note in `readme.md`.

> Only works with Node. Not in browser.

Ok, there is a tag (keynames) `node` set for this repository. But does this exclude all other platforms (browser)? I cannot build with with TypeScript. Because `require('fs')` in `build/index.js`. So it's Node only.

Personally, I would remove this dependency. And just return the ArrayBuffer or what ever. So that everyone on browser or Node can handle it.

**Comment by grimmdude** [comment]
Hi @y0nd0, 

This library _should_ work in the browser, I thought we had that `fs` dependency setup to be optional but perhaps it needs to be adjusted.

I think that's a good idea though, probably best to just return ArrayBuffer.  I'll try to get to this when I have time, but PRs are always welcome!

-Garrett

**Comment by y0nd0** [comment]
Nope, Webpack TypeScript build says "No". Maybe #71 fixes it?

**Comment by grimmdude** [comment]
Hi @y0nd0,

I've just released version 2.0.0 which removes the `Writer.saveMIDI()` function, thus removing the dependency on `fs`.  It was primarily legacy code I used to use for testing when this library first started, and seems to cause more trouble than it's worth.  There are other methods in the [`Writer`](http://grimmdude.com/MidiWriterJS/docs/Writer.html) class that can be used to get the raw MIDI data.

Thanks for bringing it up.  Hopefully this solves on your end.

-Garrett
